### DISCUSSION TITLE
Filtering packages by license

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [archlinuxcn/repo#2636] [Ideas]
hello Can you create a Linux software Repositories, all software is in line with [FOSS](https://www.gnu.org/philosophy/floss-and-foss.en.html), inevitably popular

Now, all Linux software Repositories had a serious security hazard. Packages manager Do not comply with [FOSS](https://www.gnu.org/philosophy/floss-and-foss.en.html) software,
Archlinux and manjaro official software repositories  have begun to promote closed source software and Dependencies. Such as Opera Browser. This is a very dangerous behaviour that does not open source.

**Solutions**

Filter arch Linux [packages manager](https://archlinux.org/packages/)  Hide packages by License

**Comment by oldherl** [comment]
I don't think anyone here is interested, given that Arch Linux itself isn't a full FOSS distribution. You can use Parabola if you would like a full FOSS distribution.

**Comment by DDoSolitary** [comment]
> Now, all Linux software Repositories had a serious security hazard

Closed-source/proprietary software != insecure software. Also, as @oldherl mentions, there are distributions that provide FOSS only. Please do not exaggerate.

> Archlinux and manjaro official software repositories have begun to promote closed source software and Dependencies.

Arch Linux or Manjaro has never claimed to be fully FOSS, so I think it's perfectly OK for them to provide closed-source packages. Providing a piece of software doesn't necessarily mean "promoting" it. Again, please do not exaggerate.

**Comment by hubutui** [comment]
sure, if you have resources and time.

**Comment by cuihaoleo** [comment]
Try Parabola: https://www.parabola.nu/
